packages/roosterjs-editor-types/lib/enum/EntityOperation.ts

```diff
@@ -74,4 +74,10 @@ export const enum EntityOperation {
      * This is normally happened when user undo/redo the content with an entity snapshot added by a plugin that handles entity
      */
     UpdateEntityState,
+
+    /**
+     * Notify plugins that an undo snapshot is taken while a custom logical root is used
+     * Plugins can handle this event to include entity state to include in the snapshot
+     */
+    SnapshotEntityState,
```

Comment:
No need to add this. When we receive the this new event in eventConverter, just ignore it so old plugin won't receive this event.